feature-libs/customer-ticketing/styles/components/_customer-ticketing-list.scss

```diff
@@ -0,0 +1,147 @@
+%cx-customer-ticketing-list {
+  .cx-ticketing-list-table {
+    padding: 1.5rem 0 1.125rem 0;
+    margin-bottom: 0;
+    border-top: 1px solid var(--cx-color-light);
+    border-bottom: 1px solid var(--cx-color-light);
+
+    th {
+      padding: 1.5rem 0 1.125rem 0;
+      text-align: start;
+
+      &:last-child {
+        text-align: end;
+      }
+    }
+
+    td {
+      padding: 1.625rem 0;
+
+      a {
+        min-height: 0px;
+        min-width: 0px;
+      }
+
+      @include media-breakpoint-up(md) {
+        text-align: start;
+
+        &:last-child {
+          text-align: end;
+        }
+      }
+
+      @include media-breakpoint-down(sm) {
+        width: 100%;
+        display: flex;
+        border-width: 0;
+        padding: 0 1.25rem;
+
+        &:first-child {
+          padding-top: 1.25rem;
+        }
+
+        &:last-child {
+          padding-bottom: 1.25rem;
+        }
+      }
+    }
+  }
+
+  .cx-ticketing-list-thead-mobile {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  .cx-ticketing-list-label {
+    color: var(--cx-color-secondary);
+
+    @include media-breakpoint-up(md) {
+      padding: 0 15px 0 0;
+      display: none;
+    }
+
+    @include media-breakpoint-down(sm) {
+      font-size: var(--cx-font-size, 0.875rem);
+      font-weight: var(--cx-font-weight-bold);
+      min-width: 140px;
+    }
+  }
+
+  .cx-ticketing-list-value {
+    @include media-breakpoint-down(sm) {
+      padding: 0 0 0 15px;
+      font-size: var(--cx-font-size, 1rem);
+      font-weight: $font-weight-normal;
+    }
+  }
+
+  .cx-ticketing-list-data {
+    text-align: center;
+
+    @include media-breakpoint-down(sm) {
+      text-align: start;
+    }
+  }
+
+  .cx-ticketing-list-sort {
```

Comment:
you can pull out 

```
    display: flex;
    margin: 0;
    @include media-breakpoint-down(sm) {
      flex-direction: column;
    }
```

to `.cx-ticketing-list-sort` 

as they're common to both .top and .bottom and being used for them